app/controllers/api/requests_controller.rb

```diff
@@ -1,5 +1,35 @@
 module Api
   class RequestsController < BaseController
+    include Subcollections::RequestTasks
+    include Subcollections::Tasks
+
+    def create_resource(type, _id, data)
+      assert_id_not_specified(data, type)
+
+      request_type = data.delete("request_type")
+      validate_request_type(request_type)
+
+      if data["src_id"].blank? && data["src_ids"].blank?
+        raise BadRequestError, "Must specify a resource src_id or src_ids"
+      end
+
+      requester = Hash(data.delete("requester"))
+      user = fetch_requester_user(requester)
+      auto_approve = fetch_auto_approve(requester)
```

Comment:
you're right, was using an example from MiqProvisionVirtWorkflow.  Moved that puppy out so it is its own parameter now.